src/main/java/org/folio/oaipmh/dao/PostgresClientFactory.java

```diff
@@ -36,17 +36,15 @@
 
   private static final String DEFAULT_SCHEMA_PROPERTY = "search_path";
 
-  private static final int POOL_SIZE = 5;
+  private static final int POOL_SIZE = 20;
```

Comment:
Deja vu... is this same change in multiple PRs?

Reply:
> Deja vu... is this same change in multiple PRs?
Cause i need these changes but PR still not into master, therefore I have to put it into each branch